src/pages-and-resources/discussions/app-config-form/apps/shared/DivisionByGroupFields.jsx

```diff
@@ -31,17 +32,7 @@ function DivisionByGroupFields({
             <FormSwitchGroup
               onChange={onChange}
               onBlur={onBlur}
-              className="ml-4"
-              id="allowDivisionByUnit"
-              checked={values.allowDivisionByUnit}
-              label={intl.formatMessage(messages.allowDivisionByUnitLabel)}
```

Comment:
What happened to this field?  I think we need to pause and make sure we have the right fields for this form - I'm not sure the effect of just pulling this out and not updating the rest of the data model.